rustfmt-core/rustfmt-lib/src/expr.rs

```diff
@@ -833,11 +833,8 @@ impl<'a> ControlFlow<'a> {
                 .span_after(self.span, self.connector.trim());
             let comments_span = mk_sp(comments_lo, expr.span.lo());
 
-            let missing_comments = match rewrite_missing_comment(
-                comments_span,
-                cond_shape,
-                context,
-            ) {
+            let missing_comments = match rewrite_missing_comment(comments_span, cond_shape, context)
+            {
```

Comment:
This change (https://github.com/rust-lang/rustfmt/pull/4044/commits/dbf385acdb601b9923d958cfa5ee0841c2431629) was simply to fix the misformatting here that snuck in #4055 due to temporary state of disabled tests/other formatting issues on master.

I presume it's okay to tack this on with this PR, though let me know if not!

Reply:
Yes, let's keep master green 🍏